fabric-loot-api-v2/src/main/java/net/fabricmc/fabric/api/loot/v2/LootTableEvents.java

```diff
@@ -49,9 +62,27 @@ private LootTableEvents() {
 		return null;
 	});
 
+	/**
+	 * This event can be used to replace loot tables.
+	 * If a loot table is replaced, the iteration will stop for that loot table.
+	 * This event allows access to the registries, such as enchantments.
+	 */
+	public static final Event<RegistryAwareReplace> REGISTRY_AWARE_REPLACE = EventFactory.createArrayBacked(RegistryAwareReplace.class, listeners -> (key, original, source, registries) -> {
```

Comment:
Im not a fan of thise name, however im not sure what else to suggest. Could maybe go in a whole new class?

Reply:
It might worth creating v3 module? Maybe with context object instead.